animeal/res/ka.lproj/Localizable.strings

```diff
@@ -43,7 +43,8 @@
 "action.copyIBAN"                                 = "კოპირება IBAN";
 "action.deleteAccount"                            = "Ანგარიშის წაშლა";
 "action.iWillFeed"                                = "ვაჭმევ";
-"action.finish"                                   = "";
+"action.yes"                                      = "Yes";
```

Comment:
We need Georgian lang here, currently we are using google translate rot that

Reply:
To be honest I put the same english text in Google sheet as well , for avoiding empty string at this moment))

Reply:
I will create a instruction on the kb page on how to work with string, so let's just skip it for now.